cli/io/config.go

```diff
@@ -114,32 +111,20 @@ func (config *Config) read() {
 //    "target": "http://geoffs-macbook-pro.local:8081"
 //}
 
-func (config *Config) getCredentials(target string) (username string, password string, err error) {
-	credentials, found := config.Map[credentialsKey].(map[string]interface{})
+func (config *Config) getCredentials(target string) (credentials string, err error) {
+	credentialsMap, found := config.Map[credentialsKey].(map[string]interface{})
```

Comment:
we must improve the JSON handling and parsing, but that's another PR!

Reply:
OK, lets do it, but not in this PR nor today :D. 
I'd like continue learning go, please explain me how you'd improve this and i'll try to do if in future.